fashion_police.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name = 'fashion_police'
-  spec.version = '2.4.0'
+  spec.version = '2.5.0'
```

En général, j'ai plutôt tendance à faire des releases de patch pour ce genre de modifs de dépendances, surtout si on considère que c'est un bug fix

```suggestion
  spec.version = '2.4.1'
```

T'en penses quoi ?